Makefile

```diff
@@ -1,5 +1,6 @@
 CLOUD_CONNECTOR_SCHEMA ?= https://raw.githubusercontent.com/RedHatInsights/cloud-connector/master/internal/controller/api/api.spec.json
 RBAC_CONNECTOR_SCHEMA ?= https://cloud.redhat.com/api/rbac/v1/openapi.json
+XJOIN_GRAPHQL_SCHEMA ?= https://raw.githubusercontent.com/RedHatInsights/xjoin-search/master/src/schema/schema.graphql
```

Comment:
xjoin-search can be considered deprecated. I am actively working on v2 of xjoin which will use a new, generated graphql schema. I'd prefer not to have anything new depend on xjoin-search.

Reply:
@fijshion Gotcha, do you happen to know when the v2 of xjoin will be available?  In the meantime, I'll go ahead and integrate with the inventory API

Reply:
I don't have a firm timeline. It's pretty far along but there's still a decent amount of work to be done. If I get some help on it - hopefully in the next few months.